pkgs/martinvonz/jj/registry.yaml

```diff
@@ -24,14 +24,7 @@ packages:
           - linux/amd64
           - windows/amd64
       - version_constraint: Version == "v0.3.1"
-        asset: jj_{{.Version}}_{{.Arch}}-{{.OS}}.{{.Format}}
-        format: zip
-        windows_arm_emulation: true
-        replacements:
-          amd64: x86_64
-          windows: pc-windows-gnu
-        supported_envs:
-          - windows/amd64
+        no_asset: true
```

Comment:
There is no asset.
https://github.com/martinvonz/jj/releases/tag/v0.3.1

Reply:
🤔 I'm not sure why `aqua gr` generates the wrong code.
I think this is a bug of aqua.

```console
$ aqua gr martinvonz/jj 
packages:
  - type: github_release
    repo_owner: martinvonz
    repo_name: jj
    description: A Git-compatible VCS that is both simple and powerful
    version_constraint: "false"
    version_overrides:
      - version_constraint: semver("<= 0.3.0")
        asset: jj_{{.Version}}_{{.Arch}}-{{.OS}}.{{.Format}}
        format: zip
        windows_arm_emulation: true
        replacements:
          amd64: x86_64
          linux: unknown-linux-musl
          windows: pc-windows-gnu
        checksum:
          type: github_release
          asset: jj_{{.Version}}_{{.Arch}}-{{.OS}}.zip.sha256sum
          algorithm: sha256
        overrides:
          - goos: linux
            format: tar.gz
        supported_envs:
          - linux/amd64
          - windows/amd64
      - version_constraint: Version == "v0.3.1"
        asset: jj_{{.Version}}_{{.Arch}}-{{.OS}}.{{.Format}}
        format: zip
        windows_arm_emulation: true
        replacements:
          amd64: x86_64
          windows: pc-windows-gnu
        supported_envs:
          - windows/amd64
      - version_constraint: semver("<= 0.4.0")
        asset: jj-{{.Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
        format: tar.gz
        rosetta2: true
        windows_arm_emulation: true
        replacements:
          amd64: x86_64
          darwin: apple-darwin
          linux: unknown-linux-musl
          windows: pc-windows-msvc
        overrides:
          - goos: windows
            format: zip
        supported_envs:
          - darwin
          - windows
          - amd64
      - version_constraint: Version == "v0.5.0"
        no_asset: true
      - version_constraint: semver("<= 0.13.0")
        asset: jj-{{.Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
        format: tar.gz
        rosetta2: true
        windows_arm_emulation: true
        replacements:
          amd64: x86_64
          darwin: apple-darwin
          linux: unknown-linux-musl
          windows: pc-windows-msvc
        overrides:
          - goos: windows
            format: zip
        supported_envs:
          - darwin
          - windows
          - amd64
      - version_constraint: "true"
        asset: jj-{{.Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
        format: tar.gz
        windows_arm_emulation: true
        replacements:
          amd64: x86_64
          darwin: apple-darwin
          linux: unknown-linux-musl
          windows: pc-windows-msvc
        overrides:
          - goos: darwin
            replacements:
              arm64: aarch64
          - goos: windows
            format: zip
        supported_envs:
          - darwin
          - windows
          - amd64
```

Reply:
Created the issue.

- https://github.com/aquaproj/aqua/issues/2693